packages/fast-web-utilities/src/dom.ts

```diff
@@ -1,6 +1,31 @@
 import { canUseDOM } from "exenv-es6";
 import { isBoolean } from "lodash-es";
 
+/**
+ * A test that ensures that all arguments are HTML Elements
+ */
+export function isHTMLElement(...args: HTMLElement[]): boolean {
```

Comment:
sorry - we don't want to assert these are `HTMLElement` because the point of the fn is to *determine* if they're HTML element types.

Just tried it locally, doesn't work. This should probably just be `any[]` instead of `unknown[]` since we actually don't care what the types are.